pkgs/tools/misc/valeronoi/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, boost
+, cgal
+, cmake
+, copyDesktopItems
+, gpp
+, mpfr
+, openssl
+, qtbase
+, qtimageformats
+, qtsvg
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Valeronoi";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "ccoors";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-Xa70kOPQLavuJTF9PxCgpKYj15C2fna++cFlCId0a08=";
+  };
+
+  buildInputs = [
+    gpp
+    mpfr
+    qtbase
+    qtimageformats
+    qtsvg
+  ];
+
+  nativeBuildInputs = [
+    boost
+    cgal
+    cmake
+    openssl
```

Comment:
Why do we need cgal and openssl in nativeBuildInputs?

Reply:
I've moved cgal to buildInputs, as the package does not build without this. 

It seems that building and executing the binary still works without openssl,so i have removed it.

Reply:
Did you test if the connection to the vacuum still works? Openssl probably won't be invoked before.

Reply:
I have tested this briefly and I haven't had any issues. I can still find the vacuum in my local network and connect to it. 

I'll test it further later today and let you know if everything works

Reply:
I've built and tested the program again on a different machine and haven't encountered any problems. Therefore, I think that valeronoi does not need openssl to function